src/main/scala/com/gu/CheckPriceRisePreConditions.scala

```diff
@@ -42,12 +39,6 @@ object CheckPriceRisePreConditions {
       TargetPriceRiseIsMoreThanTheCurrentPrice -> (priceRise.newPrice > currentGuardianWeeklySubscription.price),
       CurrentlyActiveProductRatePlanIsGuardianWeeklyRatePlan -> Config.Zuora.Old.guardianWeeklyProductRatePlanIds.contains(currentGuardianWeeklySubscription.productRatePlanId),
       BillingPeriodIsQuarterlyOrAnnually -> List("Annual", "Quarter").contains(currentGuardianWeeklySubscription.billingPeriod),
-      ThereDoesNotExistAFutureAmendmentOnThePriceRiseDate ->
```

Comment:
For the record: we decided that this was unnecessary since it is handled in the skip checks